### ISSUE TITLE
Right-click / context menu functions?

### ISSUE THREAD

**Issue by aikeru** [equalsraf/neovim-qt#336]
**Operating System:**
Windows 10 right this moment.
Downloaded via chocolatey.

I didn't see any other issues for this... Just wondering if there are any plans to add context menu / right-click functions to neovim-qt. I know using the mouse is kind of "anti-vim", but it seems like this would be a nice feature.

Thanks so much for making neovim-qt! :)

**Comment by pcewing** [comment]
I too would like this feature but you can set it up manually in the meantime if you want it!

Here's the guide for vim; you'll want to edit the `.reg` files to use neovim instead:
http://vim.wikia.com/wiki/Adding_Vim_to_MS-Windows_File_Explorer_Menu

**Comment by jgehrig** [comment]
I agree with @aikeru a copy-paste context menu within neovim-qt would be a nice feature. Other front-ends support such functionality.

I created a simple Pull Request to add support for the feature. #539

**Comment by jgehrig** [comment]
For the sake of reference...

To enable the context menu, add the following to `ginit.vim`:
```
nnoremap <silent><RightMouse> :call GuiShowContextMenu()<CR>
inoremap <silent><RightMouse> <Esc>:call GuiShowContextMenu()<CR>
vnoremap <silent><RightMouse> :call GuiShowContextMenu()<CR>
```

**Comment by jgehrig** [comment]
This feature is available as of `v0.2.13`.

Marking as closed.
